packages/modal/stories/ModalStory.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+  import Modal from "../Modal.svelte";
+</script>
+
+<Modal>
+  <span slot="title">Test</span>
+  <div>
+    <p>
+      This is a plain modal with no header. You can put
+      <strong>any</strong>
+      HTML content inside.
+    </p>
+    <p>
+      Click the close button or
+      <code>ESC</code>
+      to close it.
+    </p>
+  </div>
+</Modal>
```

Comment:
I you run this locally this doesn't work and I'm not sure why. The same code works in the REPL.